experimental/sys/fs.go

```diff
@@ -58,6 +58,8 @@ type FS interface {
 	//     https://pubs.opengroup.org/onlinepubs/9699919799/functions/open.html
 	OpenFile(path string, flag Oflag, perm fs.FileMode) (File, Errno)
 
+	OpenFileAt(dir File, path string, flag Oflag, perm fs.FileMode) (File, Errno)
```

Comment:
I may be mistaken, but I don't think it is really necessary to expose this to the public interface, is it? can't we just adopt it internally for now?

Reply:
You're right. Removed from interface.